pkg/vere/BUILD.bazel

```diff
@@ -140,6 +140,10 @@ cc_library(
 
 cc_binary(
     name = "urbit",
+    copts = select({
+        "@platforms//os:macos": [],
+        "//conditions:default": ["-g"],
+    }),
```

Comment:
After thinking further about this yesterday, I don't think this is the best way to disable inclusion of debug info on macOS. `-g` won't be set on macOS only when building via this target (`//pkg/vere:urbit`), whereas what we really want is to globally turn off `-g` so that all targets on macOS are built off of `-g` whether or not they were triggered by this target or not. To that end, the fix in [`b16ada6baaccdce2a3dda78caf70b55d8dbbc28e`](https://github.com/urbit/vere/tree/b16ada6baaccdce2a3dda78caf70b55d8dbbc28e) is a better one.

Reply:
Makes sense to me. Fixed.